data/sql/compound_crs.sql

```diff
@@ -586,9 +586,9 @@ INSERT INTO "compound_crs" VALUES('EPSG','9429','ETRS89 + Douglas height',NULL,'
 INSERT INTO "usage" VALUES('EPSG','14057','compound_crs','EPSG','9429','EPSG','2803','EPSG','1026');
 INSERT INTO "compound_crs" VALUES('EPSG','9430','ETRS89-GBR [OSNet v2009] + St. Marys height',NULL,'EPSG','11009','EPSG','5749',0);
 INSERT INTO "usage" VALUES('EPSG','14058','compound_crs','EPSG','9430','EPSG','2802','EPSG','1026');
-INSERT INTO "compound_crs" VALUES('EPSG','9449','ETRS89 + Malin Head height',NULL,'EPSG','4258','EPSG','5731',0);
+INSERT INTO "compound_crs" VALUES('EPSG','9449','ETRS89-IRE [IETRF2000 + Malin Head height',NULL,'EPSG','4173','EPSG','5731',0);
```

Comment:
Is this a typo in EPSG, or really there is an IETRF2000?

Reply:
looks like a typo. Should be ''ETRS89-IRE [ETRF2000] + Malin Head height'. I'll report it to IOGP (have already reported a wrong alias for SRB_ETRS89/ETRS89-SRB [STRS00]')